larq_compute_engine/tflite/kernels/utils.h

```diff
@@ -0,0 +1,17 @@
+#ifndef COMPUTE_ENGINE_TFLITE_KERNEL_UTILS_H
+#define COMPUTE_ENGINE_TFLITE_KERNEL_UTILS_H
+
+#include "tensorflow/lite/c/builtin_op_data.h"
+#include "tensorflow/lite/schema/schema_generated.h"
+
+namespace tflite {
+
+// Converts the flatbuffer activation to what is used at runtime.
+TfLiteFusedActivation ConvertActivation(ActivationFunctionType activation);
+
+// Converts the flatbuffer padding enum to what is used at runtime.
+TfLitePadding ConvertPadding(Padding padding);
```

Comment:
Not sure if we could reuse [these functions from TensorFlow](https://github.com/tensorflow/tensorflow/blob/22a73ff29265cb534d7f56a4fe7631c8f09e7813/tensorflow/lite/core/api/flatbuffer_conversions.cc#L104-L132) somehow.